http/http_service.go

```diff
@@ -269,6 +269,12 @@ func (httpSvc *HttpService) startHandler(c echo.Context) error {
 		})
 	}
 
+	if !httpSvc.cfg.CheckUnlockPassword(startRequest.UnlockPassword) {
```

Comment:
Internally it also checks, is this to return a better error message?

Reply:
yeah, instant feedback + no need to call node start

Reply:
Note: we could do the same with the Wails version. But I don't think it's that important.

Thanks!